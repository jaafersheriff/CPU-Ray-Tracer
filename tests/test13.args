firsthit spheres.pov 640 480 270 240
