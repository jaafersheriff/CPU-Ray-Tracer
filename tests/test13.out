Pixel: [270, 240] Ray: {0 0 14} -> {-0.1026 0.001036 -0.9947}
T = 13.83
Object Type: Sphere
Color: 0 0 0
